Updated Footer

Hey-Zeus-byte opened the pull request:
Hello! @fvaldez421 

Please review my code haha.

fvaldez421 commented on src/components/Footer.js:
```diff
@@ -70,32 +70,27 @@ const Footer = () => {
-              display: "flex",
-              justifyContent: "center",
-              alignItems: "baseline",
+              fontSize: "12px",
```
This should be in the styled component

fvaldez421 commented on src/components/Footer.js:
```diff
@@ -70,32 +70,27 @@ const Footer = () => {
-              display: "flex",
-              justifyContent: "center",
-              alignItems: "flex-end",
+              fontSize: "12px",
```
Same comment as above

fvaldez421 commented on src/components/Footer.js:
```diff
@@ -70,32 +70,27 @@ const Footer = () => {
             </EmailLink>
           </p>
-          <div
+          <WithinFooterFirst
```
These names are confusing. Not sure what first or second is supposed to mean

fvaldez421 commented on src/components/Footer.js:
```diff
@@ -55,41 +62,23 @@ const Footer = () => {
-              fontSize: "12px",
-            }}
-          >
+          <TextFontTypeOne>
```
This naming is a little confusing bc you have an image in it. Can we come up with a better name?

fvaldez421 commented on src/components/Footer.js:
```diff
@@ -5,11 +5,8 @@ import {ContentWrapper} from "./ContentWrapper";
-  /* border: 5px solid black; */
   display: flex;
-  flex-direction: row;
   justify-content: space-evenly;
```
lets use the `space-between` so that our gutters are always even

Hey-Zeus-byte commented:
For "Fixed footer container" I fixed the width size of the footer and updated the justify content in the following push.

![Screen Shot 2022-08-22 at 11 52 20 PM](https://user-images.githubusercontent.com/58921019/186091650-ea8ff617-a2f7-41ed-b9df-b4f31447fd3b.png)
